webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/domain/RudderDit.scala

```diff
@@ -150,6 +150,22 @@ class RudderDit(val BASE_DN:DN) extends AbstractDit {
     )
   }
 
+  def policyServerTarget(
+      uuid : String
+    , parentDN : DN
+    , name :  String = ""
+    , description : String = ""
+  ) : CATEGORY = {
```

Comment:
I don't think it should be here. Category are for the branches in DIT, like "ou=nodes", etc. Here, IIUC, you define a new entry type for leaf